### DISCUSSION TITLE
How to launch fish shell through WSL?

### DISCUSSION THREAD

**Discussion by Obspogon** [cmderdev/cmder#2701] [Q&A] [answered]
Setting `fish` as the login shell results in some issues [described here](https://fishshell.com/docs/current/index.html#default-shell) that i want to avoid.

**Comment by daxgames** [comment]
This is not really a Cmder issue/question.

Probably what the documentation you referrenced is the best way.  Thats the way I would do it on any Linux distro.

**Reply by Obspogon** [reply]
the documentation doesn't mention how to change the shell the terminal launches

**Comment by Obspogon** [comment, marked as answer]
I think I figured it out.
```
cmd /c C:\Windows\System32\wsl.exe ~ -new_console:p:%USERPROFILE%  /usr/bin/fish
```
